app/controllers/address_controller.rb

```diff
@@ -1,14 +1,4 @@
 # frozen_string_literal: true
 
 class AddressController < StandardStepsController
```

Comment:
Yes we are using this to collect address info, but `update` is defined in ` StandardStepsController` so we don't need to duplicate the logic here